internal/os/os_test.go

```diff
@@ -0,0 +1,80 @@
+package os
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_RegisterCommands(t *testing.T) {
```

Comment:
This function name should be changed to reflect the contents of the test.  This is testing the `operating-system` subcommand, so it something like TestCli_OperatingSystem` would be more accurate.